.eslintrc.js

```diff
@@ -27,7 +27,9 @@ module.exports = {
     extends: [
         "plugin:cypress/recommended",
         "eslint:recommended",
-        "plugin:import/typescript"
+        "plugin:import/typescript",
```

Comment:
Is this working when you run the lint command? @asekawa 

```
plugins: ['@wso2'], // plugin should be added like this
  extends: [
    'plugin:@wso2/typescript',
    'plugin:@wso2/strict',
    'plugin:@wso2/internal',
    'plugin:@wso2/jest',
    'plugin:@wso2/prettier'
  ],
  ```

Reply:
Also install, `eslint`, `prettier` & `jest` as dev dependencies

Reply:
Test the lint command once you do all the changes.

Reply:
These plugins were installed and tested